src/gui/sdlmain.cpp

```diff
@@ -1857,10 +1866,28 @@ Bitu GFX_SetSize(int width,
 		assert(emptytex);
 
 		memset(emptytex, 0, texture_area_bytes);
-		glTexImage2D(GL_TEXTURE_2D, 0, GL_RGBA8, texsize_w, texsize_h,
+
+		int is_framebuffer_srgb_capable;
+		SDL_GL_GetAttribute(SDL_GL_FRAMEBUFFER_SRGB_CAPABLE,
+		                    &is_framebuffer_srgb_capable);
+
+		sdl.opengl.framebuffer_is_srgb_encoded = is_sharp_shader() && is_framebuffer_srgb_capable > 0;
+
+		if (sdl.opengl.framebuffer_is_srgb_encoded)
+			LOG_INFO("OPENGL: Using sRGB framebuffer");
```

Comment:
This is a bit noisy; it gets logged twice whenever the window becomes focused. Not sure where else I can put it, any suggestions? Maybe change the log level to debug?

Reply:
I think we're assuming that this should always be the case, so maybe move it to the else side, and log it as a warning saying that the framebuffer isn't sRGB capable?

If those cases are out there, these warnings will be just as noisy, bdut hopefully those users will report this and we can learn more about the corner cases (if they exist).

Reply:
Yeah, probably that will be the best. Changed.